backend/src/controllers/metricsController.ts

```diff
@@ -41,20 +49,28 @@ const metricsController = {
       // Destructure req.body.promQLBody to get variables from the frontend.
       const { containerID, time } = req.query;
 
+      if (!time || !containerID) {
+        // error handler
+        return next();
```

Comment:
`next` function needs to be passed an error argument for it to invoke the global or default error handler: https://expressjs.com/en/guide/error-handling.html